userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright (C) 2013-2018 Draios Inc dba Sysdig.
```

Comment:
Nit: Since this is new, I'd make this just be 2019.  Ditto for other new files with new content.

Reply:
Good catch. Should we have a "Happy new year!" commit for all the other files?